app/javascript/components/Header.js

```diff
@@ -34,7 +34,11 @@ export default class Header extends Component {
               <Button variant="text">All Questions</Button>
             </NavLink>
 
-            {logged_in && <Button variant="text">My Questions</Button>}
+            {logged_in &&
+            <NavLink to={"/protectedindex"}> 
+            <Button variant="text">My Questions</Button>
```

Comment:
can you tab this line over?

Reply:
Hi Austin, I committed some small formatting changes by running prettier.